README.md

```diff
@@ -16,6 +16,11 @@ This repository holds the kcp tests that tests against the publicly available  i
   ```
 
 * Have the environment variable `KUBECONFIG` set pointing to your kcp service
+  - If your `KUBECONFIG` has multiply contexts please specific target test environment by using environment variable `E2E_TEST_CONTEXT` ( In addition, if it not exists using `kcp-stable-root` context, if `kcp-stable-root` also not exist use the `current-context` in `KUBECONFIG` instead). 
```

Comment:
s/multiply/multiple
( In addition, if it not exists using `kcp-stable-root` context, if `kcp-stable-root` also not exist use the `current-context` in `KUBECONFIG` instead) -> may be you should say something like, if the test context specified does not exist, it will try to use kcp-stable-root and if kcp-stable-root does not exist `current-context` in `KUBECONFIG` is used instead.

Reply:
Fixed.